tetris.py

```diff
@@ -177,8 +216,13 @@ def block_mover(self):
             x, y = self.current_block_center
             self.current_block_center = (x, y + 1)
 
-    def block_rotator(self, event):
-        print("TODO: should rotate")
+    def block_rotator(self):
+        '''
+        Rotates the current block
+        '''
+        self.index += 1
+        return self.get_current_block()
```

Comment:
The return value is not used when `block_rotator()` method is called, so it is not necessary:

```suggestion
```